irepela/homework_2/three_words.py

```diff
@@ -0,0 +1,11 @@
+def checkio(words: str):
```

Comment:
I would suggest for purpose of homework to improve naming of functions and add docstrigns, typings (both for input params and for return values)